lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -420,7 +420,12 @@ public SimplePageItem findCommentsToolBySakaiId(String sakaiId) {
 		DetachedCriteria d = DetachedCriteria.forClass(SimplePageItem.class).add(Restrictions.eq("sakaiId", sakaiId));
 		return (List<SimplePageItem>) getHibernateTemplate().findByCriteria(d);
 	}
-	
+
+	public List<SimplePageItem> findPageItemsByPageId(long pageId) {
```

Comment:
I would change this method to be
existsPageByPageId(long pageId)
and just return a boolean. That's more JPA like. Unless you think we'll use it elsewhere to actually get the list of pages.

Reply:
I don't think the name is appropriate, it needs the new id and that id is in the list, needs to return a list of items that belong to that page.